src/sentry/api/endpoints/auth_index.py

```diff
@@ -140,8 +159,13 @@ def put(self, request: Request):
             return Response({"detail": {"code": "ignore"}}, status=status.HTTP_403_FORBIDDEN)
 
         try:
-            # Must use the real request object that Django knows about
+            # Must use the httprequest object instead of request
```

Comment:
I paired with danny on this one and the "real request object that Django knows about" was confusing, for the django framework, it seems `Django HttpRequest` is needed instead of the `django rest framework Request`